packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -965,15 +923,15 @@ class _TableState<T> extends State<_Table<T?>> with AutoDisposeMixin {
         child: Column(
           crossAxisAlignment: CrossAxisAlignment.stretch,
           children: [
-            TableRow.tableHeader(
+            TableRow<T>.tableHeader(
               key: const Key('Table header'),
               linkedScrollControllerGroup:
                   _linkedHorizontalScrollControllerGroup,
-              columns: widget.columns,
+              columns: widget.columns as List<ColumnData<T>>,
```

Comment:
why are these type casts necessary in this constructor?

Reply:
I think because I had T? in state definition. Fixed.